.github/workflows/test.yml

```diff
@@ -62,7 +117,8 @@ jobs:
       - name: Check out
         uses: actions/checkout@v3
         with:
-          submodules: recursive
+          fetch-depth: 100
```

Comment:
Why do we have to set the fetch depth = 100 here? I think shallow clone with depth = 1 is enough.

Reply:
Fixed in https://github.com/nimblehq/kmm-templates/pull/38/commits/4845f136d9c3489313ade13983ca208df75a1c17